examples/tests/sdk_tests/dynamodb-table/global-secondary-index.test.w

```diff
@@ -0,0 +1,93 @@
+bring cloud;
+bring ex;
+bring util;
+
+let api = new cloud.Api();
+
+let table = new ex.DynamodbTable(
+    name: "blog",
+    attributeDefinitions: {
+        type: "S",
+        id: "S",
+        createdAt: "N"
+    },
+    hashKey: "type",
+    rangeKey: "id",
+    globalSecondaryIndex: [{
+        name: "CreatedAtIndex",
+        hashKey: "type",
+        rangeKey: "createdAt",
+        projectionType: "ALL"
+    }],
+);
+
+// Only Hash Key
+let table2 = new ex.DynamodbTable(
+    name: "blog2",
+    attributeDefinitions: {
+        type: "S",
+        id: "S",
+    },
+    hashKey: "type",
+    rangeKey: "id",
+    globalSecondaryIndex: [{
+        name: "TypeIndex",
+        hashKey: "type",
+        projectionType: "ALL"
+    }],
+) as "blog2";
+
+test "Global secondary index" {
+    for i in 1..11 {
+        let var id = util.nanoid();
```

Comment:
I'm not sure how stable is that, imagine a random id starts with "zz" while the last id starts with "az" (or "00" and "0z" for the first one) 
is there a way to exclude the chars "0" and "z" from the nano id? another option is to create an array of constants (will ensure that the test will run the same  way each time)